test/index.test.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+var Analytics = require('@segment/analytics.js-core').constructor;
+var integration = require('@segment/analytics.js-integration');
+var sandbox = require('@segment/clear-env');
+var tester = require('@segment/analytics.js-integration-tester');
+var Floodlight = require('../lib');
+var sinon = require('sinon');
+
+describe('DoubleClick Floodlight', function() {
+  var floodlight;
+  var analytics;
+  var options = {
+    source: '654757884637545',
+    events: [
+      {
+        key: 'Watched Westworld',
+        value: {
+          event: 'Watched Westworld',
+          cat: 'activityTag',
+          type: 'groupTag',
+          customVariable: [
+            {
+              key: 'favoriteCharacter',
+              value: 'u1'
+            },
+            {
+              key: 'episode',
+              value: 'u2'
+            }
+          ]
+        }
+      },
+      {
+        key: 'Goodbye Pablo',
+        value: {
+          event: 'Goodbye Pablo',
+          cat: 'activityTag',
+          type: 'groupTag',
+          customVariable: []
+        }
+      }
+    ]
+  };
+
+
+  beforeEach(function() {
+    analytics = new Analytics();
+    floodlight = new Floodlight(options);
+    analytics.use(Floodlight);
+    analytics.use(tester);
+    analytics.add(floodlight);
+  });
+
+  afterEach(function() {
+    analytics.restore();
+    analytics.reset();
+    floodlight.reset();
+    sandbox();
+  });
+
+  it('should have the correct settings', function() {
+    analytics.compare(Floodlight, integration('DoubleClick Floodlight')
+      .option('source', '')
+      .mapping('events'));
+  });
+
+  describe('after loading', function() {
+    beforeEach(function(done) {
+      analytics.once('ready', done);
+      analytics.initialize();
+    });
+
+    describe('track', function() {
+      beforeEach(function() {
+        analytics.spy(floodlight, 'load');
+      });
+
+      before(function() {
+        // stubbing cachebuster logic
+        sinon.stub(Math, 'random').returns(0.27005030284556764);
```

Comment:
dude I actually noticed that but was too lazy to return to normal state lol but definitely agree will make the change